### PR TITLE
Unify Polyfill.io URL across renders

### PR DESCRIPTION
**Pull request by mxdvl**
## What does this change?

- refactor(polyfillIO): extract URL for reuse
- feat(polyfillIO): add `Promise.prototype.finally`

## Why?

Currently the URL is saved as a raw string, making it hard to parse which features are enabled. Commit diff are equally hard to parse. Changes to one type of page should be shared with others. Examples of previous commits:

- ca605d1b6360ecc2b48699a7574ecc7b3cfc503b
- b62b5850f7d7effea0954ad578afdfb752b66f9b
- cd76e03844fd344418f45beccda77b1bd7fb1ba1
- 2290286a7ca266eac786df0704c5b6f6cc0a19a0
- 6d5ab3f2699c4240694a1dee6826e674920ba59a
- b18f47bb64ff79a2b06a7a5500efb8339269e69c
- ef0c08417bee26083bfe2808fd23ad4b5cfb3cd1
- 6a7fc3c2f0783050b4b2fefc328517d6ed94f228
- 2cc46046e9711255a86ec6bf5fb628fc1cfdd265
- 01289c218d3d417101c9889da88c2f2ee7138440
- 7ba088b9b234668fc6fe2f048dd58fc77e8ae05f

### CODE REVIEW

**Review by georgeblahblah — APPROVED** [review]
Much easier to see what we're including!

File: dotcom-rendering/src/lib/polyfill.io.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+/** Polyfill.io script URL configuration */
+export const polyfillIO = new URL(
```
Comment (jamesgorrie) [inline dotcom-rendering/src/lib/polyfill.io.ts:2]: Why make it a `URL` if we immediately convert it to a string?
